app/model/scoreboard.js

```diff
@@ -63,6 +65,75 @@ export default class Scoreboard {
         if(!this.isDone()) {
             this._round.nextRound();
         }
+
+        return this;
+    }
+
+    /**
+     * Mark that there were all hits this round. Assumes start with a fresh round
+     * @returns {Scoreboard} this, for chaining
+     */
+    hitAllThisRound() {
+        if(this._round.isFreshRound()) {
```

Comment:
Oh I see, this answers my question elsewhere. This is confusing to me, then...unless "Hit all" is greyed out when the round isn't fresh? That would un-confuse me.

Reply:
Added a grey out. That's been confusing when playing too--but only if i accidentally hit Hit/Miss to do a single-dart increment. I almost never use the Hit/Miss button anymore